apps/vr-tests-react-components/package.json

```diff
@@ -46,6 +46,7 @@
     "@fluentui/react-radio": "^9.0.26",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 24 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27116&commitId=7604b2cac24f80bf9a6bd317b9a9d24cac8c26a5&env=prod)

✅ There was 24 screenshots added, 0 screenshots removed, 1868 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with circle.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with rectangle.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with square - High Contrast.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|0| Added|
|Skeleton converged.Opaque Skeleton with square.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with  circle - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with  circle - High Contrast.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with  circle - RTL.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with  circle.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with rectangle - High Contrast.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with rectangle - RTL.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with rectangle.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with square - High Contrast.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with square - RTL.chromium.png|0| Added|
|Skeleton converged.Translucent Skeleton with square.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>